Rules.modular

```diff
@@ -45,7 +45,7 @@ base: $(base_pkg)
 
 modules: $(mod_pkgs)
 
-install: $(instpkg) $(appfiles)
+install: $(instpkg) $(appfiles) $(ncpath) $(ncpath_nft)
```

Comment:
I think this (and the equivalent change in `Rules.monlithic` is the only thing I disagree with. Since SECMARK is rarely used, I don't think it should be installed by default. I'd be fine with a install-netcontexts or similar new target.

Reply:
Actually, on further thought, the reason this does not exist current for modular policy is that the original intent was that modules could provide labeling fragments, like the .fc files. I think having the modular policy install net contexts is broken.

Reply:
I suppose SECMARK is not used because it's not so easy to use without good documentation and examples. The files are not read by SELinux itself but external tools like Netfilter, so perhaps they should be installed in `/usr/share/doc/selinux/examples` or something like that instead?

Reply:
That would work too. I'd be fine with an install target for docs.

Reply:
Labeling fragments for modules also make sense. Then if a module is obsoleted (or just disabled), also related labeling and port rules are removed. It would be simplest just to move all `network_port()` items to new per module labeling files. This probably would need changes to semodule, in addition to m4 machinery and gennetfilter.